OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -133,21 +139,87 @@ static int HashList<T>(List<T> xs)
 			return hash;
 		}
 
-		List<CPos> EvalPath()
+		// NOTE TO REMOVE: This previously used mobile.ToCell. actor.Location
+		// and mobile.ToCell are exactly the same always, and using
```

Comment:
I'll have a look at the property. In this case maybe a property inside Mobile with a better name would be a better fit. Maybe a CurrentLocation property inside mobile.